### PR TITLE
🤖 Simplify `hello-world` stub

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Simplify the `hello-world` exercise's stub. The goal of this is to make things easier for students to get started and to introduce as little syntax as possible.

See https://github.com/exercism/v3-launch/issues/46

### CODE REVIEW

File: exercises/practice/hello-world/lib/hello_world.ex [hunk 1]
```diff
@@ -2,8 +2,7 @@ defmodule HelloWorld do
   @doc """
   Simply returns "Hello, World!"
   """
-  @spec hello :: String.t()
```
Comment (ErikSchierboom) [inline exercises/practice/hello-world/lib/hello_world.ex:5]: I used the example implementation as the basis for this change and that did no include the `@spec` part. I'm trying to make this file as simple as possible, so I've omitted it. Is that okay?

Comment (angelikatyborska) [reply exercises/practice/hello-world/lib/hello_world.ex:5]: No, we should add it to the example implementation too. All practice exercises stubs have `@spec`s and `@doc`s.

Unfortunately our example implementations get out of date with the stub with regard to documentation. The example says:
```
  Greets the user by name, or by saying "Hello, World!"
  if no name is given
```

Which I guess used to be the case a while ago.

Comment (ErikSchierboom) [reply exercises/practice/hello-world/lib/hello_world.ex:5]: I've updated

**Review by angelikatyborska — APPROVED** [review]

**Comment by angelikatyborska** [comment]
Thanks!

**Pull request merged**